### DISCUSSION TITLE
2.3.1-realease代码无法编译通过

### DISCUSSION THREAD

**Discussion by YuanjunXu** [apache/seatunnel#4709] [General]
以下类型无法找到：
<img width="626" alt="image" src="https://user-images.githubusercontent.com/32825741/236592427-e854b0ae-2b0d-4832-adba-8808c63a6b00.png">

在 IDEA 中运行 SeaTunnelEngineServerExample ，然后编译失败，出现上图中的错误。我需要在 IDEA 中调试。

**Comment by YuanjunXu** [comment]
在 IDEA maven 中执行：clean install -DskipTests -Dcheckstyle.skip -Dmaven.javadoc.skip=true 
然后等待编译完成，就会编译通过

**Reply by zhengqiangtan** [reply]
初次编译，会遇到jdbc 连接器编译失败问题，请问有遇到过吗？
<img width="1227" alt="image" src="https://github.com/apache/seatunnel/assets/12029814/55f8b25e-f74f-4427-8fab-77df967eadb8">

**Comment by liunaijie** [comment]
run `mvn clean install`
